scripts/zkai_attention_kv_stwo_native_masked_sequence_proof_input.py

```
@@ -0,0 +1,527 @@
+        "tie_gap_bits": TIE_GAP_BITS,
+        "non_claims": NON_CLAIMS,
+        "proof_verifier_hardening": PROOF_VERIFIER_HARDENING,
+        "next_backend_step": NEXT_BACKEND_STEP,
+        "validation_commands": VALIDATION_COMMANDS,
+    }
+    for field, expected in constants.items():
+        if payload.get(field) != expected:
+            raise AttentionKvStwoNativeInputError(f"payload field mismatch: {field}")
+    journal = SOURCE.expected_journal()
+    if payload["initial_kv_cache"] != journal["initial_kv_cache"]:
+        raise AttentionKvStwoNativeInputError("initial KV cache drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)